### PR TITLE
chore(refactor codebase ): refactored codebase based on feedback

### PR DESCRIPTION
**Pull request by idowuadeleke**
#### What does this PR do?
Implements feedback gotten from LFA to the codebase

#### Description of Task to be completed?
- change status code from 422 to 400 and from 201 to 200
- Onclick banka image should redirect back to index page
- Replace account type on the index page to Role
- Console log listening port
- The sidebar should be raised on mobile view
- Remove commented code
- Add a line at end of page

#### How should this be manually tested?
User Interface can be tested y visiting the link below
https://idowuadeleke.github.io/BANKA/

Running app with npm run start to check console logged port

Using postman to test status code

#### What are the relevant pivotal tracker stories?
#165336755

#### Screenshots (if appropriate)
image
image

### CODE REVIEW

File: server/test/usertest.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ describe('Test user login and signup', () => {
     //         const { body } = res;
     //         expect(body).to.be.an('object');
     //         expect(body.status).to.be.a('number');
-    //         expect(body.status).to.be.equals(422);
+    //         expect(body.status).to.be.equals(400);
     //         expect(body.errors.isAdmin).to.be.equals('IsAdmin field is required');
     //         done();
     //       });
```
Comment (Mrshinezee) [inline server/test/usertest.js]: can u remove commented code

File: UI/CSS/style.css [hunk 2]
```diff
@@ -1266,5 +1242,5 @@ resize: none;
     border-radius:0 3px 0 0 !important;}
 
 .profiletransheader{
-  text-align:center;
+  text-align:center !important;
   border-radius:3px 3px 0 0 ;}
```
Comment (Mrshinezee) [inline UI/CSS/style.css]: add a line at the end of the page

File: server/test/usertest.js (hunk 1 above)
Comment (idowuadeleke) [reply server/test/usertest.js]: Okay

Comment (idowuadeleke) [reply server/test/usertest.js]: This has been implemented

File: UI/CSS/style.css (hunk 2 above)
Comment (idowuadeleke) [reply UI/CSS/style.css]: This has been implemented

**Pull request merged**